tests/functional/s/singledispatch_functions_py3.rc

```diff
@@ -1,3 +0,0 @@
-[testoptions]
-;https://bugs.python.org/issue10445
```

Comment:
Can you add this as a comment in the test file?

Reply:
The content of the link is "python 3.2 and 3.3 receive only security update, we can't backport the fix that is in 3.4 in it", so we can remove the link too.